core/test/fraggle-rock/scenarios/pptr-test-utils.js

```diff
@@ -38,8 +38,8 @@ function createTestState() {
 
     installSetupAndTeardownHooks() {
       before(async () => {
-        this.server = new Server();
-        this.secondaryServer = new Server();
+        this.server = new Server(0);
+        this.secondaryServer = new Server(0);
```

Comment:
Might be nice to use `createServers` here and expose a `this.servers` for pptr tests.

Reply:
createServers hardcodes the ports, so can't use that here.

Reply:
There's def. room for a better Server refactor here, but I don't wanna  get bogged down on it.